### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by pachi81**
- New complication: delta and timestamp
- Floating widget
- Fix large arrow on Galaxy Watch 5 Pro (Wear OS 3)
- Tasker action to change some settings
- Add relative time option (instead of fix timestamp)
- Trend arrow will be strikethrough instead of disappear, if there is no new value

### CODE REVIEW

**Review by froster82 — COMMENTED** [review]
Translation done. But I might have deleted to many spaces between the lines. I have the original file, so I still can paste it back, if needed.

File: mobile/src/main/res/values-pl/strings.xml [hunk 1]
```diff
@@ -67,7 +73,7 @@
     <string name="pref_color_ok">Kolor w zakresie docelowym</string>
     <string name="pref_color_ok_summary">Kolor dla wartości stęż. glukozy w zakresie docelowym.</string>
     <string name="pref_notification">Powiadomienie</string>
-    <string name="pref_notification_summary">Wyświetla stałe powiadomienie z bieżącą strzałką trendu, wartością stęż. glukozy i deltą na pasku stanu.</string>
+    <string name="pref_notification_summary">Wyświetla stałe powiadomienie z bieżącą strzałką trendu, wartością stęż. glukozy i deltą na pasku stanu.\n To powiadomienie zapobiega zatrzymaniu tej aplikacji przez system Android.</string>
```
Comment (pachi81) [inline mobile/src/main/res/values-pl/strings.xml:76]: Remove space

File: mobile/src/main/res/values-pl/strings.xml [hunk 2]
```diff
@@ -78,27 +84,30 @@
...
-    <string name="menu_group_github">GitHub</string>
+    <string name="menu_group_github" translatable="false">GitHub</string>
     <string name="menu_update">Aktualizacje</string>
-    <string name="menu_support">Wsparcie</string>
+    <string name="menu_support" translatable="false">Wsparcie</string>
     <string name="menu_help">Pomoc</string>
-
-    <string name="help_link">https://github.com/pachi81/GlucoDataHandler/blob/master/README_PL.md</string>
-
+    <string name="help_link">https://github.com/pachi81/GlucoDataHandler/blob/master/README.md</string>
```
Comment (pachi81) [inline mobile/src/main/res/values-pl/strings.xml:108]: Undo these lines -> README_PL.md and the lines with translatable="false" should not be part of this file

**Comment by pachi81** [comment]
> Translation done. But I might have deleted to many spaces between the lines. I have the original file, so I still can paste it back, if needed.

Yes, there are too many spaces removed, but no problem, I will move the strings to only have one file (not 3) and for that I will add some additional spaces and comments.

File: mobile/src/main/res/values-pl/strings.xml (hunk 1 above)
Comment (pachi81) [reply mobile/src/main/res/values-pl/strings.xml]: Done

File: mobile/src/main/res/values-pl/strings.xml (hunk 2 above)
Comment (pachi81) [reply mobile/src/main/res/values-pl/strings.xml]: Done

File: README_PL.md [hunk 3]
```diff
@@ -7,9 +7,11 @@
 
 - odbiera wartości glukozy z Juggluco
 - odbiera wartości glukozy z xDrip+
+- provides several **[widgets](#widgets)** and a floating widget for the phone
```
Comment (pachi81) [inline README_PL.md:10]: @froster82 : Please also update the new text in this file. Thank you!

Comment (froster82) [reply README_PL.md:10]: OK, got it. Will do.

**Review by froster82 — COMMENTED** [review]
Translated Readme. I worked on the English version to get it into my translation memory, but noticed some addressses for pics were different in PL. I hope I corrected them all.

**Comment by pachi81** [comment]
> Translated Readme. I worked on the English version to get it into my translation memory, but noticed some addressses for pics were different in PL. I hope I corrected them all.

Looks good, thank you! I will update the wear setting image as soon I have built the app with your translation. Thank you!

**Comment by pachi81** [comment]
@froster82: there are some new changes:
https://github.com/pachi81/GlucoDataHandler/pull/92/commits/594d3fc5c17988db9d5f94a37cb426ab4ef1ae81#diff-bd6311cc0b6875d4350bc94577a3598d016c2287731d2a5d9036cde4c9640ac6

https://github.com/pachi81/GlucoDataHandler/pull/92/commits/594d3fc5c17988db9d5f94a37cb426ab4ef1ae81#diff-53aead5c1105150732f126121569424fd9ac595640a45fc4ea67942b08ed5f6d

And also to the docu:
https://github.com/pachi81/GlucoDataHandler/pull/92/commits/783a62d20f4f79f0d6d3d1d67bf52b542868ba19
https://github.com/pachi81/GlucoDataHandler/pull/92/commits/7080dbd2ba33944d2d716dcbfe030fe7dc7c6be7

Thank you!

I also tried to move all the strings to one file, but unfortunately I got one error, I could not resolve :-( So I have to let them in several files at the moment...

**Comment by froster82** [comment]
I will try to have a look at them tomorrow. Today I was a bit preoccupied with Juggluco :) I still need to update some Polish files in the latest version.

**Comment by froster82** [comment]
I think I had to create a new pull request to complete the translation, but everything should be OK now.

**Pull request closed without merging**